src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheckTest.java

```diff
@@ -169,6 +170,7 @@ public void testNoJavadocs() throws Exception {
 
     @Test
     public void testWriteTagRecordsAndCompactCtors() throws Exception {
+        Locale.setDefault(Locale.ENGLISH);
         final String[] expected = {
```

Comment:
Please revert this change.

Please remove this Check from suppression list as all others done in their PRs for such update.

Reply:
I delete this check from the ban list "com.puppycrawl.tools.checkstyle.checks.javadoc.WriteTagCheck",  and then Locale.setDefault(Locale.ENGLISH);  needs to be set separately in each test method,  otherwise the expected error message will be in my own language, and the English error message will not pass the test.

I tried to change my compilation output to English by adding '-Duser.language=en' to idea64.exe. vmoptions, idea. properties, java compiler in idea Settings and system user variable, but it didn't work.  It can be said that my problem is stuck in that I do not know how to modify the output information of IDEA compilation into English. What do you have in mind?